fiasco/io/factory.py

```diff
@@ -58,9 +58,6 @@ class Parser(GenericParser, metaclass=ParserFactory):
     The Parser ingests the name of a raw ASCII data file and builds an
     `astropy.table.QTable` from it. A predefined parser is created based
     on the file extension, but a custom parser can also be used.
-
-    Examples
-    --------
     """
     def __init__(self, filename, custom_parser=None, **kwargs):
-        super().__init__(filename, **kwargs)
+        super().__init__(filename, custom_parser=custom_parser, **kwargs)
```

Comment:
Is this ok?

Reply:
Ok so what is happening is that at the factory level, `custom_parser` is being pulled from the `kwargs`. Instead of adding this here, the explicit keyword argument should be removed from `Parser.__init__`.